test/e2e/azure_lb.go

```diff
@@ -182,17 +183,17 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 	// TODO: determine root issue of failures of addressing the ELB from prow and fix
 	// see https://kubernetes.slack.com/archives/CEX9HENG7/p1610547551019900
 
-	//if !input.IPv6 {
-	//	By("connecting directly to the external LB service")
-	//	url := fmt.Sprintf("http://%s", elbIP)
-	//	resp, err := retryablehttp.Get(url)
-	//	if resp != nil {
-	//		defer resp.Body.Close()
-	//	}
-	//	Expect(err).NotTo(HaveOccurred())
-	//	Expect(resp.StatusCode).To(Equal(200))
-	//	Expect(err).NotTo(HaveOccurred())
-	//}
+	if !input.IPv6 {
```

Comment:
so we're _not_ skipping this for VMSS, correct?

Is the current state of things:
- for VMSS tests, run ELB test only
- for non-VMSS tests, run ILB + ELB tests

?

Reply:
100% correct. My reasoning is that ELB is the most common scenario, so if we had to pick ILB or ELB, but not both for VMSS, ELB would be my pick.